Makefile

```diff
@@ -39,7 +39,7 @@ yarn-install-dev: docker-build ## Install JS dependencies with yarn.lock file an
 
 .PHONY: watch
 watch: yarn-install-dev ## Run on http://localhost:8000 the Helpcenter, with auto refresh on file changes
-	$(DOCKER_RUN) --expose=8000 -p=8000:8000 -p=35729:35729 ${DOCKER_IMAGE_TAG} yarn gulp serve
+	$(DOCKER_RUN) --expose=8010 -p=8010:8000 -p=35729:35729 ${DOCKER_IMAGE_TAG} yarn gulp serve
```

Comment:
To avoid conflict with the localhost portal port

Reply:
Can you update the README.md or other place where the port is used/mentionned ?